ui/packages/consul-ui/app/components/consul/auth-method/search-bar/index.hbs

```diff
@@ -0,0 +1,148 @@
+            </span>
+          </BlockSlot>
+          <BlockSlot @name="options">
+    {{#let components.Optgroup components.Option as |Optgroup Option|}}
+            {{#each @filter.searchproperty.default as |prop|}}
+              <Option @value={{prop}} @selected={{contains prop @filter.searchproperty.value}}>
+                {{t (concat "common.consul." (lowercase prop))}}
+              </Option>
+            {{/each}}
+    {{/let}}
+          </BlockSlot>
+        </search.Select>
+      </search.Search>
+    </:search>
+    <:filter as |search|>
+      <search.Select
+        class="type-kind"
+        @position="left"
+        @onchange={{action @filter.kind.change}}
+        @multiple={{true}}
+      as |components|>
+        <BlockSlot @name="selected">
+          <span>
+            {{t "components.consul.auth-method.search-bar.kind.name"}}
+          </span>
+        </BlockSlot>
+        <BlockSlot @name="options">
+        {{#let components.Optgroup components.Option as |Optgroup Option|}}
+          <Option class="kubernetes" @value="kubernetes" @selected={{contains 'kubernetes' @filter.kind.value}}>Kubernetes</Option>
+          <Option class="jwt" @value="jwt" @selected={{contains 'jwt' @filter.kind.value}}>JWT</Option>
+        {{#if (env 'CONSUL_SSO_ENABLED')}}
+          <Option class="oidc" @value="oidc" @selected={{contains 'oidc' @filter.kind.value}}>OIDC</Option>
+        {{/if}}
+        {{/let}}
+        </BlockSlot>
+      </search.Select>
+      <search.Select
+        class="type-locality"
+        @position="left"
+        @onchange={{action @filter.source.change}}
+        @multiple={{true}}
+      as |components|>
+        <BlockSlot @name="selected">
+          <span>
+            {{t "components.consul.auth-method.search-bar.locality.name"}}
+          </span>
+        </BlockSlot>
+        <BlockSlot @name="options">
+      {{#let components.Optgroup components.Option as |Optgroup Option|}}
+        {{#each (array "local" "global") as |option|}}
+          <Option class="{{option}}" @value={{option}} @selected={{contains option @filter.types}}>
+            {{t (concat "components.consul.auth-method.search-bar.locality.options." option)}}
+          </Option>
+        {{/each}}
+      {{/let}}
+        </BlockSlot>
+      </search.Select>
+    </:filter>
+    <:sort as |search|>
+      <search.Select
+        class="type-sort"
+        data-test-sort-control
+        @position="right"
+        @onchange={{action @sort.change}}
+        @multiple={{false}}
+        @required={{true}}
+      as |components|>
+        <BlockSlot @name="selected">
+          <span>
+            {{#let (from-entries (array
+                  (array "MethodName:asc" (t "common.sort.alpha.asc"))
+                  (array "MethodName:desc" (t "common.sort.alpha.desc"))
+                  (array "MaxTokenTTL:desc" (t "common.sort.duration.asc"))
+                  (array "MaxTokenTTL:asc" (t "common.sort.duration.desc"))
```

Comment:
I noticed the order here might be wrong? On line 125 the sort is `desc` but the text/t is `asc`, and vice versa for line 126.